compendium/modules/w05-classes-exercise.tex

```diff
@@ -244,7 +244,7 @@
   def distanceTo(that: (Int, Int)): Double = ???
 
   /** A new Point that is moved (dx, dy) */
-  def move(dxdy: (Int, Int)): Point = ???
+  def move(dx: Int, dy: Int): Point = ???
```

Comment:
The original version isn't wrong. All points are supposed to be represented as a tuple.